lib/AppInfo/Capabilities.php

```diff
@@ -170,7 +170,7 @@ private function getCapabilitiesCircleConstants(): array {
 			'source' =>
 				[
 					'core' => [
-						Member::TYPE_USER => $this->l10n->t('Nextcloud User'),
+						Member::TYPE_USER => $this->l10n->t('Nextcloud Account'),
```

Comment:
@PVince81 @ArtificialOwl this and other instances have wrong capitalization. As per design guidelines, we use "Sentence case" not "Capital Case". So "Nextcloud account", not "Nextcloud Account" with capital A.
,